### PR TITLE
fix: escape quotes in enum string values

### LINKED ISSUE #256: Enum value with single quote causes syntax error
Hello,

I've encountered a bug where if an enum value contains a single quote, a syntax error is thrown.

```json
            "enum": [
              "",
              "Cash",
              "Insurance",
              "Insurance Out of Network",
              "Auto Accident",
              "Worker's Comp"
            ],
```

https://app.drchrono.com/openapi-schema has a download link for the schema which contains this issue.

```shell
Loading spec from .\openapi-schema.json…
SyntaxError: ')' expected. (1796:123)
  1794 |   * One of `""`, `"Cash"`, `"Insurance"`, `"Insurance Out of Network"`, `"Auto Accident"` or `"Worker's Comp"`.<br>**Note:** Patient must already have either `primary_insurance` or `secondary_insurance` or new `primary_insurance` or `secondary_insurance` is passed in request if `Insurance`, `Auto Accident` or `Worker's Comp` payment profiles are chosen.
  1795 |   */
> 1796 | "patient_payment_profile"?: ('') | ('Cash') | ('Insurance') | ('Insurance Out of Network') | ('Auto Accident') | ('Worker's Comp');
```

### PR DESCRIPTION
**Pull request by gr2m**
I've run into this problem when updating to the latest GitHub schema, this PR reproduces the problem the same way. I'm looking into fixing it

fixes #256

### CODE REVIEW

File: src/v3.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default function generateTypesV3(
           (node.enum as string[]).map((item) =>
             typeof item === "number" || typeof item === "boolean"
               ? item
-              : `'${item}'`
+              : `'${item.replace(/'/g, "\\'")}'`
```
Comment (drwpow) [inline src/v3.ts:76]: This makes sense to me. In my experience, you should never be too aggressive escaping quotes like this, as you may end up with more things escaped than anticipated.

**Review by drwpow — APPROVED** [review]

**Comment by drwpow** [comment]
I believe this fixes #256?

**Comment by gr2m** [comment]
> I believe this fixes #256?

ah yes, thanks!

**Comment by drwpow** [comment]
`2.4.1` released!

**Pull request merged**